tests/goss-common.yaml

```diff
@@ -26,12 +26,8 @@ command:
       - 27.2.0
   docker_buildx:
     exec: docker buildx version
-    exit-status: 0
-  default_java:
-    exec: java --version
-    exit-status: 0
-    stdout:
-      - 11.0.24+8
+    exit-status:
+      not: 0
```

Comment:
This test is failing on Windows: the command `docker buildx version` does exit with the exit code 0 (which was the old test case).

I'm not sure why did you change the Docker BuildX test case in this PR? Merge conflict mistake?

Reply:
It was a personal error, my bad. Let me fix it.